kodilla-stream/src/test/java/com/kodilla/stream/portfolio/BoardTestSuite.java

```diff
@@ -117,4 +126,32 @@ public void testAddTaskListFindLongTasks() {
         //Then
         Assert.assertEquals(2, longTasks);
     }
-}
\ No newline at end of file
+
+    @Test
+    public void testAddTaskListAverageWorkingOnTask() {
+        //Given
+
+        Board project = prepareTestData();
+        //When
+        List<TaskList> inProgressTasks = new ArrayList<>();
+        inProgressTasks.add(new TaskList("In progress"));
+        List<Task> tasks = project.getTaskLists().stream()
+                .filter(inProgressTasks::contains)
+                .flatMap(tl -> tl.getTasks().stream())
+                .filter(t -> t.getCreated().isBefore(LocalDate.now()))
+                .collect(toList());
+
+        long sumOfDays  = tasks.stream()
+                .map(d -> DAYS.between(LocalDate.now(),d.getCreated()))
+                .reduce((long) 0, (sum, current) -> (sum += current)) * (-1);
```

Comment:
czemu mnożysz przez -1? żeby zmienić znak? czy to nie jest przypadkiem potrzebne bo zamieniłeś kolejnością daty w DAYS.between() ? 
Jeśli już chcesz mnożyć przez -1 żeby zmienić znak to zrób to w osobnej linii bo to jest niezbyt czytelne. 

W celu zmiany znaku na dodatni warto przemyśleć zastosowanie Math.abs().